crates/cast/src/starknet_commands/account/add.rs

```diff
@@ -23,7 +23,7 @@ pub struct Add {
     pub name: String,
 
     /// Address of the account
-    #[clap(short, long)]
+    #[clap(short, long, requires = "private_key_input")]
```

Comment:
It's a bit of a hack in order to display in the help message information after "usage" that it is required to provide either `--private-key` or `--private-key-file`. This applies as well to the error message when neither of these options is provided. I'm open to suggestions for alternative ways to handle this.